### ISSUE TITLE
4 Quick Questions

### ISSUE THREAD

**Issue by ItzDash21** [sotolko/csgo-market-sniper#45]
Can you do less than a certain float or less than a certain price because for example when I type <0.03 in the config part and then run the program it gives an error whereas when I do with just a decimal/integer it works. So does anyone know a way to do less than or greater than a certain price or float?

Do you have to keep the steam browser and python open for it to work after you have ran the program or can you close them and have it run in the background?

Thirdly, how will the program buy any skins if the billing address on steam hasn't been set and filled in or do I have to do that
myself and if so how? 

Finally, it currently uses chrome can I change it to another Chromium based browser and if so how?

Any help will be very appreciated 
Thanks.

**Comment by GhostYiL** [comment]
1

> Can you do less than a certain float or less than a certain price because for example when I type <0.03 in the config part and then run the program it gives an error whereas when I do with just a decimal/integer it works. So does anyone know a way to do less than or greater than a certain price or float?
> 
> Do you have to keep the steam browser and python open for it to work after you have ran the program or can you close them and have it run in the background?
> 
> Thirdly, how will the program buy any skins if the billing address on steam hasn't been set and filled in or do I have to do that myself and if so how?
> 
> Finally, it currently uses chrome can I change it to another Chromium based browser and if so how?
> 
> Any help will be very appreciated Thanks.

1. Nope, u will have to modify the logic yourself.
2. U can set the flag to headless in the selenium, but u will have to make sure that first time auth cookies/session are saved before doing this.
3. If u check the script it simply checks the user balance field n therefore uses steam balance.
4. Sure, u can change the integrated core to firefox/edge/opera n so on, whatever is available for the webdriver, it can use.
TL.DR this repo is ded =D
>

**Comment by ItzDash21** [comment]
thanks for the reply and the answers also I've used the program a few times and it doesn't seem to buy all it says is cant find buy buttons. Do you have any idea why that could be?

**Comment by GhostYiL** [comment]
> thanks for the reply and the answers also I've used the program a few times and it doesn't seem to buy all it says is cant find buy buttons. Do you have any idea why that could be?

https://github.com/sotolko/csgo-market-sniper/issues/44#issuecomment-2106371024

**Comment by ItzDash21** [comment]
I've tried that one too but it still hasn't bought and two things on the chrome window when the program starts it goes up by one page without stopping( for example I typed 3 pages on the config and it started on page 1 then goes up page by page instead of returning to page 1 however idk if that affects the program or its just a visual bug/thing) and on python it cycles through the correct number of pages at least from what it says in this case 3 pages, however it says things like page 3 listing 670 (also this number keeps going up) which isn't possible the max should be 300.  Idk unless I have done something wrong, have you used it yourself and if so does it work for you?

**Comment by GhostYiL** [comment]
> I've tried that one too but it still hasn't bought and two things on the chrome window when the program starts it goes up by one page without stopping( for example I typed 3 pages on the config and it started on page 1 then goes up page by page instead of returning to page 1 however idk if that affects the program or its just a visual bug/thing) and on python it cycles through the correct number of pages at least from what it says in this case 3 pages, however it says things like page 3 listing 670 (also this number keeps going up) which isn't possible the max should be 300. Idk unless I have done something wrong, have you used it yourself and if so does it work for you?

Yeah I used it, however I found it to be way too slow for it's intended use, and I see no workaround with how the current way the things are set up. The numbers issue from the brief view _**get_listings_per_page_count**_ simply returns number of specified objects on page so it's either a bug so it somehow accumulates the number in total or steam changed it's limit per page. 

**TL.DR** It's pretty much pointless atm with the rate limits imposed by steam (due to their api being targeted by thousands of bots of every kind, every minute). Other then that it's a not so bad code to see how web scrapping with python works.
